src/axolotl/utils/models.py

```diff
@@ -1331,6 +1330,10 @@ def load_model(self) -> Tuple[PreTrainedModel, Optional[PeftConfig]]:
                 before_kbit_train_or_finetune=False,
             )
 
+        plugin_manager = PluginManager.get_instance()
+        plugin_manager.post_model_build(self.cfg, self.model)
```

Comment:
IMO the `post_model_build` call should come immediately after `skip_move_to_device = self.build_model(qlora_fsdp)`.

Reply:
Consider also setting `PLUGIN_MANAGER = PluginManager.get_instance()` as a module-level constant.